### PR TITLE
Task/ux quick wins/2781

### PR DESCRIPTION
**Pull request by andrea-williams**
# Changes Made:
- from Landing Page, clicking on a link to one of our apps will open the app in the same tab as the landing page
- changed the text in the header to "Email: bcws....@gov.bc.ca" instead of "Contact PSU" (this change affects the headers in all of our apps)
- added persistent underlines to the Contact (mailto) link, and to the app links in the hamburger menu of our landing page

# Test Links:
[Landing Page](https://wps-pr-2792.apps.silver.devops.gov.bc.ca/)
[MoreCast 2.0](https://wps-pr-2792.apps.silver.devops.gov.bc.ca/morecast-2)
[Percentile Calculator](https://wps-pr-2792.apps.silver.devops.gov.bc.ca/percentile-calculator)
[MoreCast](https://wps-pr-2792.apps.silver.devops.gov.bc.ca/morecast)
[C-Haines](https://wps-pr-2792.apps.silver.devops.gov.bc.ca/c-haines)
[FireBat](https://wps-pr-2792.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator)
[FireBat bookmark](https://wps-pr-2792.apps.silver.devops.gov.bc.ca/fire-behaviour-calculator?s=266&f=c5&c=NaN&w=20,s=286&f=c7&c=NaN&w=16,s=1055&f=c7&c=NaN&w=NaN,s=305&f=c7&c=NaN&w=NaN,s=344&f=c5&c=NaN&w=NaN,s=346&f=c7&c=NaN&w=NaN,s=328&f=c7&c=NaN&w=NaN,s=1399&f=c7&c=NaN&w=NaN,s=334&f=c7&c=NaN&w=NaN,s=1082&f=c3&c=NaN&w=NaN,s=388&f=c7&c=NaN&w=NaN,s=309&f=c7&c=NaN&w=16,s=306&f=c7&c=NaN&w=NaN,s=1029&f=c7&c=NaN&w=NaN,s=298&f=c7&c=NaN&w=NaN,s=836&f=c7&c=NaN&w=NaN,s=9999&f=c7&c=NaN&w=NaN)
[Auto Spatial Advisory (ASA)](https://wps-pr-2792.apps.silver.devops.gov.bc.ca/auto-spatial-advisory)
[HFI Calculator](https://wps-pr-2792.apps.silver.devops.gov.bc.ca/hfi-calculator)

### CODE REVIEW

File: web/src/features/landingPage/components/ToolCard.tsx [hunk 1]
```diff
@@ -75,13 +75,13 @@ const ToolCard: React.FunctionComponent<ToolCardProps> = (props: ToolCardProps)
     const path = props.route
     if (path.startsWith('http')) {
       return (
-        <a className={classes.link} href={path} rel="noreferrer" target="_blank">
+        <a className={classes.link} href={path} rel="noreferrer">
```
Comment (conbrad) [inline web/src/features/landingPage/components/ToolCard.tsx:78]: Why this change? We lose apps opening in new tabs with it

File: web/src/features/landingPage/components/ToolCard.tsx [hunk 2]
```diff
@@ -75,13 +75,13 @@ const ToolCard: React.FunctionComponent<ToolCardProps> = (props: ToolCardProps)
...
           {props.name}
         </a>
       )
     } else {
       return (
-        <Link className={classes.link} to={{ pathname: props.route }} target="_blank">
+        <Link className={classes.link} to={{ pathname: props.route }}>
```
Comment (conbrad) [inline web/src/features/landingPage/components/ToolCard.tsx:84]: Similar to the above

File: web/src/features/landingPage/components/ToolCard.tsx [hunk 3]
```diff
@@ -102,7 +102,6 @@ const ToolCard: React.FunctionComponent<ToolCardProps> = (props: ToolCardProps)
           size="large"
           variant="contained"
           sx={{ fontSize: '1.125rem', fontWeight: 700 }}
-          target="_blank"
```
Comment (conbrad) [inline web/src/features/landingPage/components/ToolCard.tsx:105]: Similar to above

File: web/src/features/landingPage/components/ToolCard.tsx (hunk 1 above)
Comment (andrea-williams) [reply web/src/features/landingPage/components/ToolCard.tsx:78]: It's an acceptance criteria in the ticket. BC Gov's standards for accessibility is that links shouldn't open in new tabs

Comment (conbrad) [reply web/src/features/landingPage/components/ToolCard.tsx:78]: Oof ok

**Review by conbrad — APPROVED** [review]

**Review by NicoleGovvy — APPROVED** [review]
